triqsPackages/nda/default.nix

```
@@ -32,7 +32,7 @@ stdenv.mkDerivation rec {
     openblasCompat
     ncurses
   ];
-  propagatedBuildInputs = with python3Packages; [ numpy ];
+  propagatedBuildInputs = with python3Packages; [ numpy mako scipy ];
```

Mako should be provided by `cpp2py` and is also not required to be propagated further.
```suggestion
  propagatedBuildInputs = with python3Packages; [ numpy scipy ];
```